recipes/idds-workflow/meta.yaml

```diff
@@ -9,18 +9,17 @@ source:
   url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
   sha256: cfb19c04c716beabb9c330edf976eb5a8e23c4365094c46f21f10b2875f31512
 
-build:
+build: noarch: python
   number: 0
   script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
   host:
-    - idds-common ==0.5.2
+    - idds-common =={{ version }}
```

Comment:
You likely don't need this in host per my previous comment. Only things imported by the setup.py are needed in host.

Reply:
Please see this comment